RobustPGO/outlier/OutlierRemoval.h

```diff
@@ -32,9 +32,9 @@ class OutlierRemoval {
    *  - returns: boolean of if optimization should be called or not
    */
 	virtual bool removeOutliers(const gtsam::NonlinearFactorGraph& new_factors,
```

Comment:
Weird indentation happening here. Looks like its over an indent too far.

Also move the arguments back as above.